tests/console/yast_scc.pm

```diff
@@ -16,19 +16,33 @@
 # G-Summary: harmorize zypper_ref between SLE and openSUSE
 # G-Maintainer: Max Lin <mlin@suse.com>
 
-use strict;
 use base "consoletest";
+use strict;
+use warnings;
+use utils;
 
 use testapi;
-use registration;
 
 sub run {
     select_console 'root-console';
 
     if (my $u = get_var('SCC_URL')) {
         type_string "echo 'url: $u' > /etc/SUSEConnect\n";
     }
-    yast_scc_registration;
+    script_run('yast2 scc');
+    assert_screen('yas2_scc-system_already-registered');
+    assert_screen('yas2_scc-select-extensions');
+    send_key('ret');
+    send_key_until_needle_match('yast2_scc-legacy-module-highlighted', 'down');    # should not be hardcoded
+    send_key(' ');
+    assert_screen('yast2_scc-legacy-module-selected');                             # should not be hardcoded
```

Comment:
Actually the legacy module should be fine because we also have it for SLE 12. So let's keep it but remove the comments please